protobuf/sync.proto

```diff
@@ -1,13 +1,19 @@
 syntax = "proto3";
 
 package mvds;
-option go_package = "protobuf";
+option go_package = "./;protobuf";
+
+message Offer {
+  bytes group_id = 1;
+  bytes message_id = 2;
```

Comment:
Wondering if this can be repeated bytes message_Id.
This way, if a message is for multiple groups all message_ids that sender is willing to offer can be grouped based on group_id.

Reply:
yes, that makes sense, it will save space in most cases, thank you